### DISCUSSION TITLE
02_fwi_example_NLopt

### DISCUSSION THREAD

**Discussion by venividivici666** [slimgroup/JUDI.jl#135] [Q&A] [answered]
When I try the "02_fwi_example_NLopt", the penultimate step did not run and no error was reported (figure below), I don't know why.
![image](https://user-images.githubusercontent.com/60508154/184538099-40ca1710-6110-41c3-981e-d163f6f6ecb5.png)

**Comment by mloubout** [comment]
That's odd will have a look and get back to you.

**Comment by mloubout** [comment]
There is a typo in the tutroial in cell `6`,  it should be `model0.m .= Float32.(reshape(x, model0.n))` (`.=` not `=`) and then everything runs correctly. WIll be fixed asap

**Comment by mloubout** [comment, marked as answer]
The notebook is now fixed on master and will be part of the next release.
